extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -186,10 +186,20 @@ class MongoQueueProcessor {
      * @return {undefined}
      */
     stop(done) {
-        if (!this._consumer) {
-            return setImmediate(done);
-        }
-        return this._consumer.close(done);
+        async.parallel([
```

Comment:
Add a log here that the consumer is being stopped and another log if we are skipping because there was no consumer.